### ISSUE TITLE
Load Test - Netty writeAndFlush

### ISSUE THREAD

**Issue by ffarhanaamin** [mrniko/netty-socketio#929]
Hi,

I am evaluating netty-sockio to build our next-generation real-time connect platform. I have treid to run some stress tests everything works fine until you start firing broadcast events to the connected clients.

Use Case, 

Its a very usual and simple use case.

a. User joins a room. 
b. netty-socketio sever broadcasts the join event to all the connected clients. 

Load Test

2000 clients, every 10ms new client joins the channel. When we run this test with boradcasting enabled we can see alot of load on the CPU and running the profiler we can see the contention on the following netty call

**writeAndFlush**

What is the way to workaround this? Are there any configurable parameters within the socketio server that we can tune? Is there anyway we can use [FlushConsolidationHandler to avoid writeAndFlush on every call? 

In short, we really want to scale our broadcast events to connected client, what would be the best way to move forward? 

Thanks

**Comment by mrniko** [comment]
You can add FlushConsolidationHandler in extended SocketIOChannelInitializer object through com.corundumstudio.socketio.SocketIOServer#setPipelineFactory() method. Let me know if you'll have a success with this handler.
